bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -4,6 +4,30 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
+	<channel-type id="softwareupdateavailable">
+		<item-type>Switch</item-type>
+		<label>Update Available</label>
+		<description>Car Software update available</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="softwareupdatestatus">
```

Comment:
If you know the list of available channels, you can add them here as it make it possible to translate them.

Reply:
Not really possible as the API is prone to change without notice, and not all fields are fully reverse engineered yet.

I hope for the upcoming, well-documented fleet API :)

Reply:
It is not needed to be an exhaustive list. All the known value's would do.

Reply:
Let me work on this in a separate pull request. It would need a bit of research to learn what values are well known to the Tesla developer community, which I am sure will take more than a few days.